pkg/config/validation/validation_test.go

```diff
@@ -2393,6 +2393,20 @@ func TestValidateVirtualService(t *testing.T) {
 				}},
 			}},
 		}, valid: false},
+		{name: "with no destination", in: &networking.VirtualService{
```

Comment:
These tests are not strictly required for this PR - but found these missing in VS validation. So added them.